src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,349 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  Typography,
+  Box,
+  Container,
+  Stack,
+  CssBaseline,
+  Card,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import useStyles from "./style";
+import axios from "axios";
+import Tippy from "@tippyjs/react";
+import "./style.css";
+import { breakpoints } from "../../theme/constant";
+import LinkedIn from "../../components/common/SocialMediaIcons/LinkedIn";
+import Twitter from "../../components/common/SocialMediaIcons/Twitter";
+
+/**
+ *
+ * @param {Array} arr
+ * mutates Array inplace
+ */
+function shuffleArray(arr) {
+  for (let i = arr.length - 1; i >= 0; i--) {
+    const newIndex = Math.floor(Math.random() * (i + 1));
+    [arr[i], arr[newIndex]] = [arr[newIndex], arr[i]];
+  }
+  return arr;
+}
+
+function TeamPage() {
+  useEffect(() => {
+    axios({
+      url: `https://anandpatel504.github.io/tarabai-shinde/data/ng_team.json`,
+    }).then((res) => {
+      setTeam(res.data);
+    });
+  }, []);
+  const [team, setTeam] = useState([]);
+  const [members, setMembers] = useState({
+    teamMembers: true,
+    volunteers: false,
+  });
+  const classes = useStyles();
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
+  const isActiveIpad = useMediaQuery("(max-width:1300px)");
+  // console.log(breakpoints.values.sm);
+  console.log(isActive);
+  function Popup(props) {
+    return (
+      <div className={classes.team_descriptionPopup}>
+        {/* <div className="d-flex align-items-center"> */}
+        <div className={classes.team_popupDetails}>
+          <Typography variant="subtitle1" className={classes.team_cardTitle}>
+            {props.Name}
+          </Typography>
+          {props.linkedin ? (
+            <a
+              href={props.linkedin}
+              target="_blank"
+              className={classes.team_socialIcon}
+            >
+              <LinkedIn />
+            </a>
+          ) : (
+            <></>
+          )}
+          {props.twitter ? (
+            <a
+              href={props.twitter}
+              target="_blank"
+              className={classes.team_socialIcon}
+            >
+              <Twitter />
+            </a>
+          ) : (
+            <></>
+          )}
+        </div>
+
+        <Typography
+          variant="body1"
+          style={
+            props.Content === "Awaiting content from team member"
+              ? { color: "grey" }
+              : {}
+          }
+          paragraph
+        >
+          {props.Content}
+        </Typography>
+      </div>
+    );
+  }
+  return (
+    <>
+      <CssBaseline />
+      <div>
+        <Container
+          maxWidth="lg"
+          style={
+            isActive ? { padding: 0, marginTop: "24px" } : { marginTop: "40px" }
+          }
```

Comment:
You can also use [responsive values](https://mui.com/material-ui/react-grid/#responsive-values) instead of `style` for the margins.  But for padding, this might not work as intended as setting it to `0` for `xs` and not `sm` means it'll always be `0`.
```suggestion
          mt={{ xs: "24px", sm: "40px" }}
          style={ isActive ? { padding: 0 } : { } }
```